submitted_models/cerberus_rmf_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,592 @@
+                                <dynamic_bias_correlation_time>300.0</dynamic_bias_correlation_time>
+                            </noise>
+                        </y>
+                        <z>
+                            <noise type="gaussian">
+                                <mean>0</mean>
+                                <stddev>1e-2</stddev>
+                                <bias_mean>0.1</bias_mean>
+                                <bias_stddev>0.001</bias_stddev>
+                                <dynamic_bias_stddev>0.002</dynamic_bias_stddev>
+                                <dynamic_bias_correlation_time>300.0</dynamic_bias_correlation_time>
+                            </noise>
+                        </z>
+                    </linear_acceleration>
+                </imu>
+            </sensor>
+            <sensor name="air_pressure" type="air_pressure"> 
+                <always_on>1</always_on> 
+                <update_rate>20</update_rate> 
+                <air_pressure> 
+                    <reference_altitude>0</reference_altitude> 
+                    <noise type="gaussian">
+                        <mean>0.00000008</mean>
+                    </noise>
+                </air_pressure>
+            </sensor>
+            <sensor name="magnetometer" type="magnetometer">
+                <always_on>1</always_on>
+                <update_rate>20</update_rate>
+                <magnetometer>
+                    <x>
+                        <noise type="gaussian">
+                            <mean>0.000000080</mean>
+                            <bias_mean>0.000000400</bias_mean>
+                        </noise>
+                    </x>
+                    <y>
+                        <noise type="gaussian">
+                            <mean>0.000000080</mean>
+                            <bias_mean>0.000000400</bias_mean>
+                        </noise>
+                    </y>
+                    <z>
+                        <noise type="gaussian">
+                            <mean>0.000000080</mean>
+                            <bias_mean>0.000000400</bias_mean>
+                        </noise>
+                    </z>
+                </magnetometer>
+            </sensor>
+
+            <!-- <visual name="camera_visual_right">
+                <pose>0.08 -0.032 0.10 0 -0.261799 0</pose>
+                <geometry>
+                    <box>
+                        <size>0.02 0.025 0.025</size>
+                    </box>
+                </geometry>
+            </visual> -->
+
+            <sensor name="camera_front_right" type="camera">
+                <pose>0.08 -0.032 0.10 0 -0.261799 0</pose>
+                <always_on>1</always_on>
+                <update_rate>30</update_rate>
+                <camera name="camera_front">
+                    <horizontal_fov>2.8</horizontal_fov>
+                    <lens>
+                        <intrinsics>
+                            <fx>285.727813720703</fx>
```

Comment:
These focal length intrinsics don't line up with our general formula:

`fx = fy = width / ( 2 * tan (hfov / 2 ) )`  which equals:  73.13

Is this intended, or from using actual camera calibration parameters?

Reply:
CC: @mihirk284

Reply:
I additionally went ahead and zeroed out the distortion parameters, because they aren't simulated in Ignition, and propagating them to ROS's camera_info could cause people to interpret the images as distorted.

Reply:
@mjcarroll @nkoenig  thank you for iterating, and apologies for the delay.
The camera calibration parameters are taken from the actual sensor onboard the RMF robot.

I will quickly reverify them since they seem a bit off

Reply:
@mihirk284 Did you verify the camera calibration parameters match the ones used on the RMF robot?